packages/storage/src/indexed-db/config.ts

```diff
@@ -2,4 +2,4 @@
  * The version number for the IndexedDB schema. This version number is used to manage
  * database upgrades and ensure that the correct schema version is applied.
  */
-export const IDB_VERSION = 49;
+export const IDB_VERSION = 50;
```

Comment:
question: what happens if we don't bump this version? The changes seem to be backwards compatible and would work without resync... or not?